example-app/src/main/java/com/adyen/checkout/example/service/ExampleDropInService.kt

```diff
@@ -56,7 +56,7 @@ class ExampleDropInService : DropInService() {
             val paymentRequest = createPaymentRequest(
                 paymentComponentData = paymentComponentJson,
                 shopperReference = keyValueStorage.getShopperReference(),
-                amount = keyValueStorage.getAmount(),
+                amount = state.data.amount,
```

Comment:
Good catch! I think we have this issue all around our example app, reading the stored amount rather than the actual amount returned from the component. In most cases it's not a problem but with gift cards it can be indeed